### DISCUSSION TITLE
MQTT v5.0 support

### DISCUSSION THREAD

**Discussion by sushantAtAws** [rabbitmq/rabbitmq-server#3049] [Questions] [answered]
Hi, 
The home readme says that v3.1 is supported. Is the support for v5.0 in works? When is it estimated to be delivered?

**Comment by michaelklishin** [comment, marked as answer]
MQTT v5.0 is not supported. Our team does not provide ETA estimates. This is open source software you get for free. You are welcome to contribute what you find missing. A good place to start would be to compile a list of meaningful protocol changes between those versions.

**Reply by sushantAtAws** [reply]
Thanks for the reply Michael.
